RibbonPhotos/Services/ImagesListService.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ImagesListService.swift
+//  RibbonPhotos
+//
+//  Created by Julia Romanenko on 23.01.2023.
+//
+
+import Foundation
+
+final class ImagesListService {
+    static let shared = ImagesListService()
+    static let didChangeNotification = Notification.Name(rawValue: "ImagesListServiceDidChange")
+    
+    private (set) var photos: [Photo] = []
```

Comment:
общепринятый стиль такой 
```suggestion
    private(set) var photos: [Photo] = []
```